ddl/ddl_tiflash_api.go

```diff
@@ -115,8 +122,10 @@ type TiFlashManagementContext struct {
 	TiFlashStores             map[int64]helper.StoreStat
 	HandlePdCounter           uint64
 	UpdateTiFlashStoreCounter uint64
-	UpdateMap                 map[int64]bool
+	PollCounter               uint64
+	ProgressCache             map[int64]string
```

Comment:
Why string rather than float64 here?

Reply:
> Why string rather than float64 here?

Because the `progress` saved in ETCD is string, so it's no need to update the ETCD if the string has no change.